tla-io/src/main/scala/at/forsyte/apalache/io/lir/PrettyWriter.scala

```diff
@@ -297,14 +297,6 @@ class PrettyWriter(
             text("{") <> nest(line <> binding <> ":" <> nest(line <> filter)) <> line <> text("}")
         ) ///
 
-      // a function of multiple arguments that are packed into a tuple: don't print the angular brackets <<...>>
-      case OperEx(op @ TlaFunOper.app, funEx, OperEx(TlaFunOper.tuple, args @ _*)) =>
-        val argDocs = args.map(exToDoc(op.precedence, _, nameResolver))
-        val commaSeparatedArgs = folddoc(argDocs.toList, _ <> text(",") <@> _)
-        group(
-            exToDoc(TlaFunOper.app.precedence, funEx, nameResolver) <> brackets(commaSeparatedArgs)
-        ) ///
-
```

Comment:
Why do we have to remove this? In TLA+, `f[x, y]` is a short-form for `f[<<x, y>>]`. Look at this example: https://gist.github.com/konnov/ce0ca9071dbe75b936e96dbd99224f0d

Reply:
It does work like that for tuples, but not for sequences:
```tla
--------------------------------- MODULE test ---------------------------------
EXTENDS Integers, Sequences

VARIABLES
  f

Init ==
  f = [ l \in {<<1>>, <<>>, <<1, 2>>} |-> Len(l) ]

Next ==
  UNCHANGED f

Inv ==
  f[<<1>>] = 1

Inv2 ==
  f[1] = 1

=============================================================================
```

TLC fails to check Inv2 here: 
```
Error: Attempted to check equality of the function << >> with the value:
1
While working on the initial state:
f = (<<>> :> 0 @@ <<1>> :> 1 @@ <<1, 2>> :> 2)

Error: TLC was unable to fingerprint.
```

I believe there is no way to disambiguate between tuples and sequences at this point, so the only option was to remove it completely.

Reply:
Interesting. I think the short-form syntax only applies to pairs, triples, etc.

Reply:
Yeah, these invariants work for your example:

```tla
Inv ==
  f[<<1, 2>>] = 2

Inv2 ==
  f[1, 2] = 2
```

Reply:
Ah interesting. Yeah, the problem I hit was on a list of one element (from the hashes you wrote in ICS23 btw).